gatsby_fp/src/components/Layout/Body/Fields/Education.js

```diff
@@ -3,44 +3,38 @@ const shortid = require("shortid")
 
 export default function Education({ educationList }) {
 
+  const liInnerHTMLList =
```

Comment:
The most widely used camelCase variant by far seems to be https://google.github.io/styleguide/javaguide.html#s5.3-camel-case which would make this `liInnerHtmlList`